exporter/metrics.go

```diff
@@ -80,27 +80,44 @@ func (e *Exporter) registerConstMetricGauge(ch chan<- prometheus.Metric, metric
 }
 
 func (e *Exporter) registerConstMetric(ch chan<- prometheus.Metric, metric string, val float64, valType prometheus.ValueType, labelValues ...string) {
-	descr := e.metricDescriptions[metric]
-	if descr == nil {
-		descr = newMetricDescr(e.options.Namespace, metric, metric+" metric", labelValues)
-	}
+	description := e.findOrCreateMetricDescription(metric, labelValues)
 
-	if m, err := prometheus.NewConstMetric(descr, valType, val, labelValues...); err == nil {
+	if m, err := prometheus.NewConstMetric(description, valType, val, labelValues...); err == nil {
 		ch <- m
 	}
 }
 
 func (e *Exporter) registerConstSummary(ch chan<- prometheus.Metric, metric string, labelValues []string, count uint64, sum float64, latencyMap map[float64]float64, cmd string) {
-	descr := e.metricDescriptions[metric]
-	if descr == nil {
```

Comment:
same

Reply:
By "same", do you refer to the rebase comment, or that this should be reverted?